v20.2/disaster-recovery.md

```diff
@@ -0,0 +1,333 @@
+toc: true
+---
+
+While CockroachDB is built to be fault-tolerant and to recover automatically, sometimes disasters happen. A _disaster_ is any event that puts your cluster at risk, and usually means your cluster is experiencing [hardware failure](#hardware-failure), [data failure](#data-corruption), or has [compromised security keys](#compromised-security-keys). Having a disaster recovery plan enables you to recover quickly, while limiting the consequences.
+
+## Hardware failure
+
+When planning to survive hardware failures, these are the minimum replication factors to apply in order to have greater resilience.
+
+{{site.data.alerts.callout_info}}
+Increasing data replication factors may have an impact on [transaction](transactions.html) performance, since quorum levels are increasing.
+{{site.data.alerts.end}}
+
+### Single-region survivability planning
+
+The table below displays the minimum replication factor needed to survive simultaneous hardware failures (e.g., disk, node, availability zone (AZ), etc.) for a single-region CockroachDB cluster:
+
+<table>
+  <thead>
+    <tr>
+      <th>Simultaneous Failure</th>
+      <th>3 nodes <br>(3 AZ/Racks)</th>
+      <th>4 nodes <br>(4 AZ/Racks)</th>
+      <th>5 nodes <br>(5 AZ/Racks)</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td style="color:#46a417"><b>Disk</b></td>
+      <td>Rep Factor = 3</td>
+      <td>Rep Factor = 3</td>
+      <td>Rep Factor = 3</td>
+    </tr>
+      <td style="color:#46a417"><b>Node</b></td>
+      <td>Rep Factor = 3</td>
+      <td>Rep Factor = 3</td>
+      <td>Rep Factor = 3</td>
+    <tr>
+      <td style="color:#46a417"><b>AZ/Rack</b></td>
+      <td>Rep Factor = 3</td>
+      <td>Rep Factor = 3</td>
+      <td>Rep Factor = 3</td>
+    </tr>
+    <tr>
+      <td style="color:#46a417"><b>2 Nodes</b></td>
+      <td>Rep Factor = 5</td>
+      <td>Rep Factor = 5</td>
+      <td>Rep Factor = 5</td>
+    </tr>
+    <tr>
+      <td style="color:#46a417"><b>AZ/Rack + Node</b></td>
+      <td>Rep Factor = 9</td>
+      <td>Rep Factor = 7</td>
+      <td>Rep Factor = 5</td>
+    </tr>
+    <tr>
+      <td style="color:#46a417"><b>2 AZ/Racks</b></td>
+      <td>Not possible</td>
+      <td>Not possible</td>
+      <td>Rep Factor = 5</td>
+    </tr>
+  </tbody>
+</table>
+
+For example, a single-region, 3-node cluster would need a minimum replication factor of 3 to survive a disk failure.
+
+### Single-region recovery
+
+For hardware failures in a single-region cluster, the recovery actions vary and depend on the type of infrastructure used (e.g., virtual machines, Kubernetes, bare metal).
+
+For example, consider a CockroachBD cluster with the following setup:
+
+- Single-region
+- 5 nodes
+- Each node is in a separate availability zone or rack
+- Replication factor of 5
+
+The table below describes what actions to take to recover from various hardware failures in this example cluster:
+
+<table>
```

Comment:
Table formatting with the hover is weird :( I know this is difficult to wrangle...